mkdocs.yml

```diff
@@ -76,6 +76,7 @@ nav:
 - 'Design docs': 
     - 'git-submodules': 'design/git-submodules.md'
     - 'git-submodule-storage': 'design/git-submodule-storage.md'
+    - 'Introducing JJ run': 'design/run.md'
```

Comment:
Just `jj run` might be a better label here.

Reply:
done